storage/basedb/storage.go

```diff
@@ -14,6 +15,14 @@ type Options struct {
 	Ctx       context.Context
 }
 
+// ITxn interface for badger transaction like functions
+type ITxn interface {
```

Comment:
you can remove the starting `I` (`ITxn` > `Txn`)

`IDb` is there from historical reasons..